### PR TITLE
Wizard recipe: zimg-v3.0.1

### PR DESCRIPTION
**Pull request by innerlee**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: zimg
* Version: v3.0.1

@staticfloat please review and merge.

### CODE REVIEW

**Comment by innerlee** [comment]
The error might related to https://stackoverflow.com/questions/31242106/fp16-type-undefined-in-gnu-arm-c

image

**Comment by innerlee** [comment]
CI seems to have problems. The pending one is finished actually.

File: Z/zimg/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+./autogen.sh
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [p for p in supported_platforms() if p["arch"]!="aarch64"]
```
Comment (giordano) [inline Z/zimg/build_tarballs.jl:25]:
```suggestion
platforms = filter!(p -> arch(p) != "aarch64", supported_platforms())
```
What's the problem with aarch64?

File: Z/zimg/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+products = [
+    LibraryProduct("libzimg", :libzimg)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
```
Comment (giordano) [inline Z/zimg/build_tarballs.jl:38]: Are you ok to build this for the new platforms (armv6l-linux-* and aarch64-apple-darwin) but require Julia v1.6?

Comment (innerlee) [reply Z/zimg/build_tarballs.jl:38]: Sure, I'm ok

File: Z/zimg/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+./autogen.sh
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter!(p -> arch(p) != "aarch64", supported_platforms())
```
Comment (giordano) [inline Z/zimg/build_tarballs.jl:25]:
```suggestion
platforms = filter!(p -> arch(p) != "aarch64", supported_platforms(; experimental=true))
```

File: Z/zimg/build_tarballs.jl (hunk 2 above)
Comment (giordano) [reply Z/zimg/build_tarballs.jl:38]:
```suggestion
build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

**Pull request merged**